TypeCobol.Test/Parser/Preprocessor/DirectiveResultFiles/Insert.txt

```diff
@@ -17,8 +17,9 @@
 [24,29:000003]<IntegerLiteral>{3}
 
 -- Line 6 --
-*** DIRECTIVE INSERT 1 ([1,6:insert]<INSERT>[8,13:000001]<IntegerLiteral>{1}) ***
-[14,20:-000010]<IntegerLiteral>{-10}
+*** DIRECTIVE INVALID SYNTAX -> INSERT 0 ([1,6:insert]<INSERT>) ***
+[8,20:000001-000010]<UserDefinedWord>
+Line 6[8,20] <27, Error, Syntax> - Syntax error : mismatched input '000001-000010' expecting integer literal
```

Comment:
Statement at line 6 seems syntactically correct. Is this a limitation introduced by this PR ?